.github/workflows/test.yml

```
@@ -0,0 +1,23 @@
+name: Kubernetes Javascript Client - Validation
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '10', '8' ]
```

Any chance we can add ['12]

```suggestion
        node: [ '12', '10', '8' ]
```

As this is the latest LTS versions.

---

done (removed '8')